package-lock.json

```diff
@@ -3507,12 +3507,14 @@
         "balanced-match": {
           "version": "1.0.0",
           "bundled": true,
-          "dev": true
+          "dev": true,
```

Comment:
These changes to the `package-lock.json` come from [a bug](https://npm.community/t/package-lock-json-keeps-changing-between-platforms-and-runs/1129) in the `npm`, and thus should not be committed.